src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/tests/test_rdbms_commands.py

```diff
@@ -374,6 +374,78 @@ def _test_log_file_mgmt(self, resource_group, server, database_engine):
 
         self.assertIsNotNone(result[0]['name'])
 
+    def _test_vnet_mgmt(self, resource_group, server, database_engine):
+        rg = resource_group
+        vnet_rule_1 = 'rule1'
+        vnet_rule_2 = 'rule2'
+
+        # Create vnet's - vnet1 and vnet2
+
+        vnetName1 = 'vnet1'
+        vnetName2 = 'vnet2'
+        subnetName = 'subnet1'
+        addressPrefix = '10.0.1.0/24'
+        endpoint = database_engine
+
+        # Vnet 1
+        self.cmd('network vnet create -g {} -n {}'.format(rg, vnetName1))
+        self.cmd('network vnet subnet create -g {} --vnet-name {} -n {} --address-prefix {} --service-endpoints {}'
+                 .format(rg, vnetName1, subnetName, addressPrefix, endpoint),
+                 checks=JMESPathCheck('serviceEndpoints[0].service', endpoint))
```

Comment:
Instead of using JMESPatchCheck directly, use this:
```Python
checks=self.check('serviceEndpoints[0].service', '{endpoint}')
```

This requires that "endpoint" is registerd in your kwargs.